openml/runs/functions.py

```diff
@@ -349,82 +361,93 @@ def _run_task_get_arffcontent(model, task, class_labels):
     X, Y = task.get_X_and_y()
     arff_datacontent = []
     arff_tracecontent = []
-    user_defined_measures = defaultdict(lambda: defaultdict(dict))
+    user_defined_measures_fold = defaultdict(lambda: defaultdict(dict))
+    user_defined_measures_sample = defaultdict(lambda: defaultdict(lambda: defaultdict(dict)))
```

Comment:
We now have measures per fold (dict with 3 levels of keys) and measures per sample (dict with 4 levels of keys).